### ISSUE TITLE
AesCrypto.encrypt is not a function.

### ISSUE THREAD

**Issue by ashish-dhiman** [rocwangv/react-native-aes-kit#6]
I am creating a react native project with expo.
When I try to run the sample code, I encounter the said error "[Unhandled promise rejection: TypeError: AesCrypto.encrypt is not a function. (In 'AesCrypto.encrypt(plaintxt, secretKey, iv)', 'AesCrypto.encrypt' is undefined)]".

**Comment by Rotarepmi** [comment]
Any updates to that issue?

**Comment by michelegoh** [comment]
Have you [ejected your app](https://docs.expo.io/versions/latest/expokit/eject/)? I don't think this works on Expo projects.

**Comment by Rotarepmi** [comment]
Yup it won't work on Expo. You have to use plain react-native or ExpoKit.
While using ExpoKit you have to publish the project first and then you can use `./android/gradlew installDebug`.
Should work fine :)
